### PR TITLE
rolling should now check if you've landed on a card spot

### PR DESCRIPTION
**Pull request by gphoon**

### CODE REVIEW

File: src/cs414/a5/groupA/monopoly/server/GameServiceImpl.java [hunk 1]
```diff
@@ -311,7 +311,30 @@ public Token handleRoll(Token currentPlayer) { // TODO return TokenActionWrapper
...
-			if (getDeedOwner(currentPlayer.getGameId(), currentPlayer.getPosition()) == null) {
+			if (checkForCardSpot(currentPlayer.getPosition())){
+				Card c = getCard();
+				int type = c.getType();
+				switch (type) {
+					case 1: // get paid
+						currentPlayer.setMoney(currentPlayer.getMoney() + c.getAmount());
+						break;
+					case 2: // have to pay
+						if (currentPlayer.getMoney() - c.getAmount() >= 0)
```
Comment (Brad--) [inline src/cs414/a5/groupA/monopoly/server/GameServiceImpl.java]: There should be an else case for this I think

Comment (gphoon) [inline src/cs414/a5/groupA/monopoly/server/GameServiceImpl.java]: i think it will be handled different really soon

Comment (gphoon) [inline src/cs414/a5/groupA/monopoly/server/GameServiceImpl.java]: maybe we shouldn't merge this

**Comment by gtfreymuth** [comment]
We decided to implement this in a different way in a different branch

**Pull request closed without merging**